EuclideanGeometry/Foundation/Axiom/Basic/Plane.lean

```diff
@@ -47,35 +47,75 @@ instance : AddTorsor (Vec) P := by infer_instance
 
 /- vector $AB +$ point $A =$ point $B$ -/
 @[simp]
-theorem start_vadd_vec_eq_end (A B : P) : (VEC A B) +ᵥ A = B := sorry
+theorem start_vadd_vec_eq_end (A B : P) : (VEC A B) +ᵥ A = B := by
+  rw [Vec.mk_pt_pt]
+  exact vsub_vadd B A
 
 @[simp]
-theorem vadd_eq_self_iff_vec_eq_zero {A : P} {v : Vec} : v +ᵥ A = A ↔ v = 0 := sorry
+theorem vadd_eq_self_iff_vec_eq_zero {A : P} {v : Vec} : v +ᵥ A = A ↔ v = 0 := by
+  constructor
+  intro h
+  have k : v +ᵥ A -ᵥ A = A-ᵥ A := by
+    rw [h]
+  have u : v +ᵥ A -ᵥ A = v := by
+    simp
+  rw [u] at k
+  simp at k
+  exact k
+  intro h
+  rw[h]
+  simp
```

Comment:
It is possible to write this proof in a shorter way. Please read https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/AddTorsor.html# and find the correct theorems. One way is exactly `zero_vadd`. the other way can be also reduced in length using correct theorems.